streams/src/main/java/org/apache/kafka/streams/state/internals/MeteredKeyValueStore.java

```diff
@@ -45,12 +45,14 @@
  * @param <K>
  * @param <V>
  */
-public class MeteredKeyValueStore<K, V> extends WrappedStateStore.AbstractStateStore implements KeyValueStore<K, V> {
+public class MeteredKeyValueStore<K, V>
+    extends WrappedStateStore.AbstractStateStore
+    implements KeyValueStore<K, V> {
 
     private final KeyValueStore<Bytes, byte[]> inner;
-    private final Serde<K> keySerde;
-    private final Serde<V> valueSerde;
-    private StateSerdes<K, V> serdes;
+    final Serde<K> keySerde;
+    final Serde<V> valueSerde;
+    StateSerdes<K, V> serdes;
```

Comment:
Need to make package private, because it's set in child class (cf `#initStoreSerde()`)

Reply:
:+1: Thanks for the explanation.